pkg/agent/cluster.go

```diff
@@ -136,7 +142,14 @@ func (czero *Cluster) IsBootstrapComplete() (bool, bool, error) {
 		logrus.Trace("Current API Status: Agent Rest API: down, Bootstrap Kube API: down")
 		if !czero.installHistory.RestAPISeen && !czero.installHistory.ClusterKubeAPISeen {
 			logrus.Debug("Agent Rest API never initialized. Bootstrap Kube API never initialized")
-			logrus.Info("Waiting for cluster install to initialize. Sleeping for 30 seconds")
+			elapsedSinceInit := time.Since(czero.installHistory.ClusterInitTime)
+			// After allowing time for the interface to come up, check if Node0 can be accessed via ssh
+			if elapsedSinceInit > 2*time.Minute && !czero.CanSSHToNodeZero() {
+				logrus.Info("Cannot access Node0. There may be a network configuration problem, check console for additional info")
```

Comment:
For consistency I think it'd be better to use "rendezvous host" instead of node0

Reply:
Fixed